vividus/src/main/java/org/vividus/bdd/transformer/AbstractTableLoadingTransformer.java

```diff
@@ -47,19 +55,78 @@
                 .filter(not(String::isBlank))
                 .collect(Collectors.toList());
 
-        Stream<String> examplesTablesStream = tables.stream();
+        Stream<DescriptiveTableWrapper> examplesTablesStream = IntStream.range(0, tables.size())
```

Comment:
Not necessary when allowEmptyTables = true ?

Reply:
Imho its reasonable price to keep the code straightforward